DESCRIPTION

```diff
@@ -17,7 +17,9 @@ Imports:
   psych,
   qgraph,
   reshape2,
-  semPlot
+  semPlot,
+  GPArotation,
+  textshaping
```

Comment:
can you remove `textshaping` again? this is not a dependency of the module, right?